src/statmodels.jl

```diff
@@ -199,6 +201,13 @@ function show(io::IO, ct::CoefTable)
             str[i,pvc] = format_pvc(mat[i,pvc])
         end
     end
+    for ic in ct.integercols
+        if ic != 0
+            for i in 1:nr
+                str[i,ic] = @sprintf("%u",mat[i,ic])
```

Comment:
`%i` is more appropriate, as the integer might be signed (though currently it doesn't make any difference). Anyway, I think a better solution is `sprint(showcompact,Int(mat[i,ic]))`: this is consistent with the code used for other columns, and if for some reason the value isn't an integer, an error will be raised, instead of rounding silently.